systest/ludicrous/run_test.go

```diff
@@ -1,7 +1,8 @@
+//go:build standalone
```

Comment:
again, I have found a lot of these. I can tag them all if it's really necessary

Reply:
^ +1